docs/modules/getting-started/pages/start-ee-cluster.adoc

```diff
@@ -0,0 +1,160 @@
+= Start a Local Enterprise Cluster
```

Comment:
This page is unnecessary. I think we can remove it. We already have this tutorial in 'Start a local cluster'.